opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -357,6 +357,11 @@ class Span(trace_api.Span):
             this `Span`.
     """
 
+    def __new__(cls, *args, **kwargs):
+        if cls is Span:
+            raise TypeError("Span cannot be instantiated directly")
```

Comment:
Can we make this more helpful? May be mention that user should use tracer instead of even link to tracer/getting-started documentation?